src/test/deploy/functions/release/fabricator.spec.ts

```diff
@@ -840,9 +875,33 @@ describe("Fabricator", () => {
         { platform: "gcfv2" }
       );
 
-      await fab.updateV2Function(ep);
+      await fab.updateV2Function(ep, new scraper.SourceTokenScraper());
       expect(run.setInvokerUpdate).to.not.have.been.called;
     });
+
+    it("tries to grab new token on abort", async () => {
+      const sc = new scraper.SourceTokenScraper();
+      sc.poller({
+        metadata: {
+          sourceToken: "magic token",
+          target: "projects/p/locations/l/functions/f",
+        },
+      });
+
+      gcfv2.updateFunction.onFirstCall().rejects({ message: "unknown" });
+      gcfv2.updateFunction.resolves({ name: "op", done: true });
```

Comment:
q: shouldn't this be `onSecondCall()`? Otherwise, I'd expect the `updateFunction` mock's `onFirstCall()` to be overwritten?